### ISSUE TITLE
Auto enable filters and columns based on vcf header

### ISSUE THREAD

**Issue by dnil** [robinandeer/puzzle#170]
Would be nifty to check at least header (best existence of actual tag for any variant) to see what filters and display columns to enable. Empty ones, or filters for non-annotated properties, make little sense.

**Comment by moonso** [comment]
Question is what to look for...

**Comment by dnil** [comment]
Precisely! #162?

**Comment by moonso** [comment]
Yes I know, we have tried to go down that road before and i'm a bit scared for that...

**Comment by dnil** [comment]
What went wrong then? Anything we can mitigate?
Options? Keep code dependent on particular annotation programs and versions ranges, and implement different annotator conventions as needed in various places, while keeping the number of things to read at a minimum?

**Comment by dnil** [comment]
Ideally, annotation programs would reach a concensus on field names in INFO ANN/CSQ..
